src/main/java/net/sf/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -30,9 +30,12 @@ public void readJournalListFromResource(String resourceFileName) {
     }
 
     public void readJournalListFromFile(File file) throws FileNotFoundException {
-        readJournalList(new FileReader(Objects.requireNonNull(file)));
+        try(FileReader fr = new FileReader(Objects.requireNonNull(file))) {
+            readJournalList(fr);
+        } catch(IOException e) {
+            LOGGER.info("Could not read journal list from file " + file.getAbsolutePath());
+        }
```

Comment:
Oh yeah, this I had a bit of confusion with:

Removing the catch block leads to an error as IOException is not caught. Removing the try completely (as in the original code) leads to a warning about an warning that there is a potential resource leak.

What about catching the IOException and throw a FileNotFoundException in the catch part?